docs/api/scheduler.md

```diff
@@ -1361,6 +1361,60 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Node utilization
+
+Show how every node is distributed with regard to dominant resource utilization.
+
+**URL** : `/ws/v1/scheduler/node-utilization`
+
+**Method** : `GET`
+
+**Auth required** : NO
+
+### Success response
+
+**Code** : `200 OK`
+
+**Content examples**
+
+```text
```

Comment:
it should be json

Reply:
Hi @chia7712 , I will fix in [YUNIKORN-2316](https://issues.apache.org/jira/browse/YUNIKORN-2316).

Reply:
merge into master, @chenyulin0719 please fix this trivial comment in the follow-up.